doc/user_guide/datamodels.md

```diff
@@ -0,0 +1,65 @@
+Representing a datamodel (entity)
+----------------------------------
+
+The (concepts)[concepts.md] describes the underlying structure of DLite datamodels.
+
+Here, at set of rules on how to create a datamodel is presented.
+
+Note that several other possibilities are avilable, and this can be seen in the
+examples and tests present in the repository. 
+
+We choose here to present only one method as mixing repsentation methods might 
+be confusing. Note, however that yaml and json representations are interchangable.
+
+A generic example with some comments for clarity can be seen below.
+```yaml
+uri: http://namespace/version/name
+description: This describes how to make a yaml representation of a datamodel
+dimensions: # Simplest to represent as a dict, set to {} if no dimensions
+  name_of_dimension: description of dimension
+properties:
+  name_of_property1:
+    description: What is this property
+    type: ref # Can be any on string, float, double, int, ref ....
+    unit: unit # can be ommitted, not relevant with type ref
+    shape: [name_of_dimension] # ommit if shape is 1
+    $ref:  http://namespace/version/name_of_referenceddatamodel # only if type is ref
+```
+
+A slightly more realistic example is the "Person" entity, where we want to describe his/her name, age and skills.
+
+```yaml
+uri: http://onto-ns.com/meta/0.1/Person
+description: A person.
+dimensions: 
+  N: Number of skills.
+properties:
+  name:
+    description: Full name.
+    type: string
+  age:
+    description: Age of person.
+    type: float
+    unit: years
+  skills:
+    description: List of skills.
+    type: string
+    shape: [N]
+```
+
+First we have "uri" identifying the entity, and a human description.
```

Comment:
Suggest to use backticks (`) for keywords. They gives better highlighting in the rendered text.

If you accept this suggestion, you should also replace the quotes below with backticks.

```suggestion
First we have `uri` identifying the entity, and a human description.
```